simexpal/launch/common.py

```diff
@@ -453,9 +468,16 @@ def close(self):
 		# Consume any output that might be ready.
 		events = sel.select(timeout=1)
 		for (sk, mask) in events:
-			if not sk.data.progress():
+			if sk.fileobj == signal_reader:
+				sel.unregister(signal_reader)
+				child.kill()
+				do_exit = True
```

Comment:
Why is this needed? After the child is killed, our usual event loop logic should notice that the process terminated and proceed correctly, shouldn't it?

Reply:
We also need to terminate the parent process. Otherwise experiments that where started through the same ``simex e launch`` command will be launched. I added a comment to this line to make it clear